docs/teachertool/catalog-shared.json

```diff
@@ -11,6 +11,18 @@
             "use": "custom_function_called",
             "template": "A custom function exists and gets called",
             "docPath": "/teachertool"
+        },
+        {
+            "id": "76D86387-E3CF-4BEB-B62C-B811F5997631",
+            "use": "block_comment_used",
+            "template": "At least one comment was left on a block in the code",
```

Comment:
Perhaps: "At least one block of code has comments"?

I wish we could just say "Program contains at least one comment" but I understand the hesitation since it doesn't actually check for workspace comments yet. Would be great if we could figure that out and update this accordingly.

Reply:
Or even shorter: "At least one block has comments"